hpOneView/resources/resource.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, con, uri):
         self._uri = uri
         self._task_monitor = TaskMonitor(con)
 
-    def build_query_uri(self, start=0, count=-1, filter='', query='', sort='', view='', fields='', uri=None):
+    def build_query_uri(self, start=0, count=-1, filter='', query='', sort='', view='', fields='', scope_uris='', uri=None):
```

Comment:
Yep, as I mentioned before, the tests were breaking because the way they were being called.
This call was the one failing: 
```python
self.resource_client.get_all(1, 500, filter, query, sort, view, 'name,owner,modified', scope_uris)
```
It is actually passing in the arguments in their declaration order, so that means the contents `scope_uris` would be assigned to `uri` inside the `get_all` method, and the `scope_uris` inside the `get_all` would be left empty.

To get around this, you moved the `scope_uris` inside the method one position to the left, making "uri" be in the last position again, and that first call work. 
This is, however, a breaking change exactly because of that. 

Also, after adding in scope_uris as the last argument, the call could already work if made as:
```python
self.resource_client.get_all(1, 500, filter, query, sort, view, 'name,owner,modified', scope_uris=scope_uris)
```
since when you pass in `<key>=<value>`, python understands that you are not just passing arguments in sequential order.

All in all, I'm fine either way, with `scope_uris` as the last argument or as a breaking change, but I find it important to call attention to this and make sure this change is **desired** and not just something that happened.